### PR TITLE
Fix external links - 1th iteration (#30697)

### PR DESCRIPTION
**Pull request by Captain1653**
References #30697

I like small PR. Is it acceptable?

Kind regards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,10 +30,10 @@ You can join these groups and chats to discuss and ask Akka related questions:
 
 In addition to that, you may enjoy following:
 
-- The [news](http://akka.io/news) section of the page, which is updated whenever a new version is released
-- The [Akka Team Blog](http://blog.akka.io)
+- The [news](https://akka.io/news) section of the page, which is updated whenever a new version is released
+- The [Akka Team Blog](https://blog.akka.io)
```
Comment (Captain1653) [inline README.md:34]: `https://akka.io/news` and `https://blog.akka.io` redirect to the same page `https://akka.io/blog/`

File: README.md [hunk 2]
```diff
@@ -17,8 +17,8 @@ Learn more at [akka.io](https://akka.io/).
 Reference Documentation
 -----------------------
 
-The reference documentation is available at [doc.akka.io](http://doc.akka.io),
-for [Scala](http://doc.akka.io/docs/akka/current/scala.html) and [Java](http://doc.akka.io/docs/akka/current/java.html).
+The reference documentation is available at [doc.akka.io](https://doc.akka.io),
+for [Scala](https://doc.akka.io/docs/akka/current/scala.html) and [Java](https://doc.akka.io/docs/akka/current/java.html).
```
Comment (Captain1653) [inline README.md:21]: Scala link redirects to `https://doc.akka.io/docs/akka/current/?language=scala`
Java link redirects to  `https://doc.akka.io/docs/akka/current/?language=java`

**Slash** before **question** in the URL. Is it ok?

Comment (johanandren) [reply README.md:21]: Yep, that's fine.

File: README.md (hunk 1 above)
Comment (johanandren) [reply README.md:34]: News: https://akka.io/blog/news-archive.html
Blog: https://akka.io/blog/article-archive.html

**Review by johanandren — APPROVED** [review]

**Review by octonato — APPROVED** [review]
Yes, small PR is acceptable. 😄 
Thanks @Captain1653

**Pull request merged**